frontend/dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:21
```

Comment:
is it working with node 21? Also we don't need a docker file for frontend. Don't add this in docker-compose

Reply:
ok i  will revert back to backend and , compose to only setup mongo and backend.